projects/sso/src/index.html

```diff
@@ -7,6 +7,8 @@
 
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link rel="icon" type="image/x-icon" href="favicon.ico">
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Noto+Sans&display=swap">
```

Comment:
Can we self-host these so we don't need to ping Google servers all the time?

Reply:
Perhaps, but I would not consider that in the scope of this project